components/eamxx/src/physics/mam/eamxx_mam_optics_process_interface.hpp

```diff
@@ -0,0 +1,84 @@
+#ifndef EAMXX_MAM_OPTICS_HPP
+#define EAMXX_MAM_OPTICS_HPP
+
+#include <share/atm_process/atmosphere_process.hpp>
+#include <share/util/scream_common_physics_functions.hpp>
+#include <share/atm_process/ATMBufferManager.hpp>
+
+#include <ekat/ekat_parameter_list.hpp>
+#include <ekat/ekat_workspace.hpp>
+#include <ekat/logging/ekat_logger.hpp>
+#include <mam4xx/mam4.hpp>
+
+#include <string>
+
+#ifndef KOKKOS_ENABLE_CUDA
+#define protected public
```

Comment:
This is a bit too hacky, I think. `protected` and `private` are two C++ keywords, so I would recommend against redefining those for the CPP inside a header file. You could use alternative names, such as `ProtectedExceptCuda` or something along that line.

Reply:
Fair enough--I'll do it.